docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_grafana/_index.md

```diff
@@ -0,0 +1,187 @@
+
+3. User is verifying the successful deployment of Log Analytics workspace, Azure Managed Grafana, AMA extension, and data collection rules creation.
+
+## Deployment Option 1: Azure portal
+
+- Click the <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoft%2Fazure_arc%2Fmain%2Fazure_arc_servers_jumpstart%2Fgrafana%2Fgrafana-template.json" target="_blank"><img src="https://aka.ms/deploytoazurebutton"/></a> button and enter values for the the ARM template parameters.
+
+  > **NOTE:** Use the following command to get the AAD id of the current signed in user and copy it to the Grafana Adminprincipal Id parameter:
+
+  ```shell
+  az ad signed-in-user show --query id -o tsv
+  ```
+
+  ![Screenshot showing Azure portal deployment with required parameters](./03.png)
+
+  ![Screenshot showing successful Azure portal deployment](./04.png)
+
+## Deployment Option 2: ARM template with Azure CLI
+
+As mentioned, this deployment will leverage ARM templates.
+
+- Clone the Azure Arc Jumpstart repository
+
+    ```shell
+    git clone https://github.com/microsoft/azure_arc.git
+    ```
+
+- Edit the [_parameters file_](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/grafana/grafana-template.parameters.json) providing the values that match your configuration as described above.
+
+    ![Screenshot of ARM template parameters file](./05.png)
+
+- Locate the [_ARM template_](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/grafana/grafana-template.json) and deploy it by running the following command:
+
+    ```shell
+    az deployment group create --resource-group <Name of the Azure resource group> \
+    --template-uri "https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/grafana/grafana-template.json" \
+    --parameters <grafana-template.parameters.json parameter file location>
+    ```
+
+- Once the template has completed its run, you should see an output as follows:
+
+    ![Screenshot of ARM template execution output](./06.png)
+
+- You will have the Azure Monitor Agent (AMA) deployed on your Windows or Linux system and reporting to the Log Analytics workspace that has been created. You can verify by going back to your Azure Arc-enabled server, **Extensions** section:
+
+    ![Screenshot of AMA extension on Windows](./07.png)
+
+- Moreover, a **Data Collection Rule (DCR)** is created to send logs from the Azure Arc-enabled servers to the new **Log Analytics workspace**. In addition, the **Azure Managed Grafana** service is deployed which we will explore later in this scenario.
+
+    ![Screenshot of Data Collection Rules and Log Analytics workspace](./08.png)
+
+- Note that the template deployment has also created a required role assignment at resource group level. It will be used by **Azure Managed Grafana** to be able to access your monitoring data from its **Managed Identity**:
```

Comment:
```suggestion
- Note that the template deployment has also created a required role assignment at the resource group level. It will be used by **Azure Managed Grafana** to be able to access your monitoring data from its **Managed Identity**:
```